tox.ini

```diff
@@ -0,0 +1,53 @@
+# content of: tox.ini , put in same dir as setup.py
+[tox]
+envlist = py38, style, coverage, docs, py37, py36, docstyle
+
+
+[testenv]
+deps =
+    ipdb
+    pytest
+    scipy
+    textdistance
+commands =
+    pytest tests/ {posargs}
+
+
+[testenv:style]
+skip_install = True
+usedevelop = False
+deps = flake8
+commands =
+    flake8 setup.py galaxychop tests --ignore=E402 {posargs}
+
+
+'''[testenv:coverage]
+deps =
+    coverage
+    pytest-cov
+    textdistance
+passenv = 
+    TRAVIS    
+commands =
+    - coverage erase
+    pytest -q tests/ --cov=galaxychop --cov-append --cov-report=
+    coverage report --fail-under=80 -m
+
+
+[testenv:docstyle]
+deps = pydocstyle
+commands =
+    pydocstyle galaxychop --convention=numpy
+
+
+[testenv:docs]
+description = "Invoke sphinx-build to build th HTML docs"
+whitelist_externals =
+    make
+usedevelop = True
+skip_install = False
+changedir = docs
+deps =
+    -r {toxinidir}/docs/requirements.txt
+commands =
+    sphinx-build -W -b html -d {envtmpdir}/doctrees source {envtmpdir}/html'''
```

Comment:
Is this to make it commented?

Reply:
Yes, but I thought I could do it the same way as Python. I erased it.